include/bluetooth/audio/vcs.h

```diff
@@ -0,0 +1,504 @@
+ * @param inst   Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_aics_type_get(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Read the Audio Input Control Service input status.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ * @param inst   Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_aics_status_get(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Mute the Audio Input Control Service input.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ * @param inst   Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_aics_unmute(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Unmute the Audio Input Control Service input.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ * @param inst   Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_aics_mute(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Set input gain to manual.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ * @param inst   Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_aics_manual_gain_set(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Set the input gain to automatic.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ * @param inst   Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_aics_automatic_gain_set(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Set the input gain.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ * @param inst   Pointer to the Audio Input Control Service instance.
+ * @param gain   The gain in dB to set (-128 to 127).
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_aics_gain_set(struct bt_conn *conn, struct bt_aics *inst,
+			 int8_t gain);
+
+/**
+ * @brief Read the Audio Input Control Service description.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ * @param inst   Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_aics_description_get(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Set the Audio Input Control Service description.
+ *
+ * @param conn          Connection to peer device, or NULL to read local server
+ *                      value.
+ * @param inst          Pointer to the Audio Input Control Service instance.
+ * @param description   The description to set.
```

Comment:
Is the description copied here as well?

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/34314#discussion_r617449778